compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -2179,6 +2179,15 @@ impl<'tcx> TyS<'tcx> {
         }
     }
 
+    /// Get the `i`-th element of a tuple.
```

Comment:
This was discussed here: https://rust-lang.zulipchat.com/#narrow/stream/219381-t-libs/topic/Accessing.20a.20field.20in.20a.20tuple